AsyncDisplayKit/Details/ASFlowLayoutController.mm

```diff
@@ -90,10 +90,9 @@ - (NSSet *)indexPathsForScrolling:(ASScrollDirection)scrollDirection rangeMode:(
     currPath.row++;
 
     // Once we reach the end of the section, advance to the next one.  Keep advancing if the next section is zero-sized.
-    while (currPath.row >= [(NSArray *)completedNodes[currPath.section] count] && currPath.section < completedNodes.count - 1) {
+    while (currPath.row >= [(NSArray *)completedNodes[currPath.section] count] && currPath.section < endPath.section) {
       currPath.row = 0;
       currPath.section++;
-      ASDisplayNodeAssert(currPath.section <= endPath.section, @"currPath should never reach a further section than endPath");
```

Comment:
Why remove this?  Esp. if it was added after the original authoring of the code, I think I may have added it when encountering issues from ASTableViewStressTest etc